pkg/runner/options/options.go

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package options
+
+type Options struct {
+	SchemaVersion string // SPDX Version
+	Indent        int
```

Comment:
What is Indent for?

Reply:
It's an option that controls the indentation of the json output. It is more of a demo to show how we're wiring the options down to the deepest objects. We can drop i t later if we need to.